lib/commands/connect/export.js

```diff
@@ -15,17 +15,21 @@ module.exports = {
   needsApp: true,
   needsAuth: true,
   run: cli.command(co.wrap(function* (context, heroku) {
-    let connection = yield api.withConnection(context, heroku);
-    let url = '/api/v3/connections/' + connection.id + '/actions/export';
-    let response = yield api.request(context.auth.password, 'GET', url)
+    let connection, response;
```

Comment:
option 3 doesn't work either then because it will still print the cli.action info.  Maybe the --json flag implies stdout and no  decoration?